lib/npg_pipeline/runfolder_scaffold.pm

```diff
@@ -45,6 +48,22 @@ sub create_product_level {
     # can use in lane-level data, it will produce output and will overwrite this file.
     #
     $self->_create_tileviz_lane_indexes();
+
+    #####
+    # In per-product no_archive directories, which were created above,
+    # create links to cram output of stage1 if the links do not yet exist. 
+    #
+    # A link is created without an error even if its target does not exit.
+    # The target will be created during the p4stage1 step.
+    #
+    my $ext = 'cram';
+    foreach my $p ( @{$self->products->{'data_products'}}) {
+      my $link   = $p->file_path($p->stage1_out_path($self->no_archive_path()), ext => $ext);
+      if (!-l $link) { # a link might exist if we are re-using the analysis directory
+        my $target = $p->file_path($self->recalibrated_path(), ext => $ext);
+        symlink $target, $link or croak "Failed to create a symlink $link";
```

Comment:
Could we consider using  `abs2rel` here so the symlinks aren't broken when the runfolder is moved (e.g. when moving to outgoing),